src/components/Modal.tsx

```diff
@@ -10,61 +9,64 @@ import labels from '../constants/labels.json'
 import { StackNavigationProp } from '@react-navigation/stack'
 import { RoutesParamsType } from '../navigation/NavigationTypes'
 import { RouteProp } from '@react-navigation/native'
+import styled from 'styled-components/native'
 
-export const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center'
-  },
-  overlay: {
-    marginTop: 0,
-    backgroundColor: COLORS.lunesOverlay
-  },
-  modal: {
-    backgroundColor: COLORS.white,
-    alignItems: 'center',
-    justifyContent: 'center',
-    width: wp('85%'),
-    borderRadius: 4,
-    position: 'relative',
-    paddingVertical: 31
-  },
-  closeIcon: {
-    position: 'absolute',
-    top: 8,
-    right: 8,
-    width: 24,
-    height: 24
-  },
-  message: {
-    textAlign: 'center',
-    fontSize: wp('5%'),
-    color: COLORS.lunesGreyDark,
-    fontFamily: 'SourceSansPro-SemiBold',
-    width: wp('60%'),
-    marginBottom: 31,
-    paddingTop: 31
-  },
-  lightLabel: {
-    color: COLORS.lunesWhite,
-    fontSize: wp('4%'),
-    fontWeight: '600',
-    textAlign: 'center',
-    textTransform: 'uppercase',
-    fontFamily: 'SourceSansPro-SemiBold',
-    letterSpacing: 0.4
-  },
-  darkLabel: {
-    color: COLORS.lunesBlack,
-    fontSize: wp('4%'),
-    fontWeight: '600',
-    textAlign: 'center',
-    textTransform: 'uppercase',
-    fontFamily: 'SourceSansPro-SemiBold',
-    letterSpacing: 0.4
-  }
-})
+const ModalContainer = styled.Modal`
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+`;
+const Container = styled.View`
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+    margin-top: 0;
+    background-color: ${COLORS.lunesOverlay};
+`;
+const ModalStyle = styled.View`
+    background-color: ${COLORS.white};
+    align-items: center;
+    justify-content: center;
+    width: ${wp('85%')};
+    border-radius: 4;
+    position: relative;
+    padding-top: 31;
+    padding-bottom: 31;
+`;
+const CloseIconStyle = styled.TouchableOpacity`
+    position: absolute;
+    top: 8;
+    right: 8;
+    width: 24;
+    height: 24;
+`;
+const Message = styled.Text`
+    text-align: center;
+    font-size: ${wp('5%')};
+    color: ${COLORS.lunesGreyDark};
+    font-family: 'SourceSansPro-SemiBold';
+    width: ${wp('60%')};
+    margin-bottom: 31;
+    padding-top: 31;
+`;
+const LightLabel = styled.Text`
```

Comment:
here it ould make sense to have a generic class as Light and DarkLabel share all attributes but color. Or use props for setting the color.